stdlib/public/core/Unicode.swift

```diff
@@ -140,260 +227,61 @@ public struct UTF8 : UnicodeCodec {
     let bit1 = (lut1 >> index) & 1
 
     switch (bit1, bit0) {
-    case (0, 0): // 2-byte sequence.
+    case (0, 0): // 2-byte sequence, [ XXX XXX CU1 CU0 ].
```

Comment:
It's the byte layout of buffer, i.e. the code units (CU0 & CU1) are in the lower 2 bytes and the upper two (the XXXs) are garbage.